apps/openmw/engine.cpp

```diff
@@ -140,6 +138,7 @@ void OMW::Engine::frame(float frametime)
             mEnvironment.getMechanicsManager()->update(frametime,
                 guiActive);
         }
+        mEnvironment.getWorld()->markCellAsUnchanged();
```

Comment:
I did that because I wanted to clear the player's `mHitAttemptActor` (I've now renamed it `mHitAttemptActorId)` when changing cells. Otherwise a follower in the new cell could still target an enemy from an earlier cell that attacked the player. When I checked `MWBase::Environment::get().getWorld()->hasCellChanged(),` though, it was always returning false, and when I traced it back to engine.cpp, I saw that `markCellAsUnchanged()` was being called before actors were updated, so that's why I reordered it.